layout/base.html

```diff
@@ -14,19 +14,24 @@
       href="https://fonts.googleapis.com/css2?family=Inter:wght@100..900&display=swap"
       rel="stylesheet"
     />
+    <style>
```

Comment:
Why not place these instructions in our academy.css stylesheet?

Reply:
I removed the font-family links for Inter in Edly Academy because we are not using it.

Reply:
My comment didn't refer to the Inter font-family links, but the `.hs-form ...{...}` rules below. I think they should be moved to the academy.css file.

Reply:
Yes, I also removed it.